implementation.md

```diff
@@ -1106,3 +1106,380 @@ The following best practice is derived from the [VAST 4.2 spec](https://iabtechl
+
+To calculate the total cost of the transaction:
+(AUCTION_PRICE / 1000) * AUCTION_MULTIPLIER = COST
+(2.50 / 1000) * 30.3 = $0.07575
+
+The above auction would translate to an invoice of $0.07575.
+
+	
+### 7.9.5 - DOOH Example Scenarios
+
+#### 7.9.5.1 - DOOH Banner Bid Request
+
+```javascript
+{
+    "$schema": "https://raw.githubusercontent.com/knittingmedia/OpenRTB/main/Schema/OpenRTB_OOH_V1.json",
+    "id": "162059897743978051070",
+    "at": 1,
+    "imp": [
+        {
+            "id": "007",
+            "secure": 1,
+            "exp": 360,
+            "bidfloor": 5.0,
+            "bidfloorcur": "GBP",
+            "banner": {
+                "w": 1080,
+                "h": 1920,
+                "format": [
+                    {
+                        "w": 1080,
+                        "h": 1920
+                    },
+                    {
+                        "w": 720,
+                        "h": 1366
+                    }
+                ],
+                "mimes": [
+                    "image/jpg",
+                    "image/png",
+                    "text/html"
+                ]
+            },
+            "pmp": {
+                "private_auction": 0,
+                "deals": [
+                    {
+                        "id": "123",
+                        "at": 1,
+                        "bidfloor": 4.50
+                    },
+                    {
+                        "id": "333",
+                        "at": 1,
+                        "bidfloor": 3.00
+                    },
+                    {
+                        "id": "444",
+                        "at": 1,
+                        "bidfloor": 2.00
+                    }
+                ]
+            },
+            "dt": 12345670,
+            "etime": 10,
+            "qty": {
+                "multiplier": 14.2,
+                "sourcetype": 1,
+                "vendor": "route.org.uk"
+            }
+        }
+    ],
+    "device": {
+        "geo": {
+            "lat": 0.0001,
+            "long": -0.1235,
+            "lastfix": 0
+        },
+        "devicetype": 8,
+        "w": 1080,
+        "h": 1920,
+        "ppi": 72,
+        "ifa": "1235461904",
+        "ifa_type": "ppid",
+        "eids": [
+            {
+                "source": "oohspace.co.uk",
+                "uids": [
+                    {
+                        "atype": 501,
+                        "id": "1235461904"
+                    }
+                ]
+            }
+        ]
+    },
+    "dooh": {
+        "content": {},
+        "id": "15790",
+        "keywords": "Roadside, Billboard, D96",
+        "name": "This",
+        "publisher": {
+            "id": "G1",
+            "name": "Global"
+        },
+        "domain": "global.com",
+        "venuetax": 1,
+        "venuetypeid": 30101
```

Comment:
Did `venuetypeid` change to `venuetype`? I just see `venuetype` in the spec table. There a few references to `venuetypeid` in the examples.